scene/2d/skeleton_2d.cpp

```diff
@@ -268,18 +661,120 @@ void Skeleton2D::_notification(int p_what) {
 			_update_transform();
 		}
 
+		set_process(true);
+		set_physics_process(true);
 		request_ready();
 	}
 
 	if (p_what == NOTIFICATION_TRANSFORM_CHANGED) {
 		RS::get_singleton()->skeleton_set_base_transform_2d(skeleton, get_global_transform());
 	}
+	if (p_what == NOTIFICATION_PROCESS) {
+		if (modification_stack.is_valid()) {
+			execute_modifications(get_process_delta_time(), SkeletonModificationStack2D::EXECUTION_MODE::execution_mode_process);
+		}
+	}
+	if (p_what == NOTIFICATION_PHYSICS_PROCESS) {
```

Comment:
Usually we use `NOTIFICATION_INTERNAL_PHYSICS_PROCESS`, that can be activated using `set_physics_process_internal(true)`.